### PR TITLE
Register reaverstudio.is-a.dev

### PR DESCRIPTION
**Pull request by playreaver**
Register reaverstudio.is-a.dev with CNAME record pointing to https://playreaver.github.io/website/.

### CODE REVIEW

File: domains/playreaver.is-a.dev
Comment (JustDeveloper1) [inline domains/playreaver.is-a.dev:1]: you should delete this file

**Review by justdevw — COMMENTED** [review]
you need to delete playreaver.is-a.dev file

File: domains/playreaver.json
Comment (JustDeveloper1) [inline domains/playreaver.json:1]: cname must be playreaver.github.io

File: domains/reaverstudio.json
Comment (JustDeveloper1) [inline domains/reaverstudio.json:1]: cname must be playreaver.github.io

**Review by justdevw — COMMENTED** [review]
cname must be `playreaver.github.io`

**Review by iostpa — CHANGES_REQUESTED** [review]
Heads up that you can't connect 2 domains to one github repo, please choose one of them.

**Comment by playreaver** [comment]
> Heads up that you can't connect 2 domains to one github repo, please choose one of them.

done

**Review by iostpa — APPROVED** [review]
Seems good to me (used translator and didn't find any weird parts)

**Pull request merged**